disk_objectstore/cli.py

```diff
@@ -183,3 +184,73 @@ def optimize(
         container.clean_storage(vacuum=vacuum)
     size = sum(f.stat().st_size for f in dostore.path.glob("**/*") if f.is_file())
     click.echo(f"Final container size: {round(size/1000, 2)} Mb")
+
+
+@main.command("backup")
+@click.argument("dest", nargs=1, type=click.Path())
+@click.option(
+    "--keep",
+    default=1,
+    help="Number of previous backups to keep in the destination. (default: 1)",
+)
+@click.option(
+    "--rsync_exe",
```

Comment:
Convention is to not use underscores in CLI parameters
```suggestion
    "--rsync-exe",
```